src/store/orders/ordersSlice.tsx

```diff
@@ -38,51 +46,103 @@ export const ordersSlice = createSlice({
     },
   },
   extraReducers: (builder) => {
+    builder.addCase(createOrder.pending, (state) => {
+      state.orderStatus = "loading";
+    });
     builder.addCase(createOrder.fulfilled, (state, action) => {
-      state.orders.push(action.payload);
+      state.orderStatus = "completed";
+    }),
+      builder.addCase(createOrder.rejected, (state) => {
+        state.orderStatus = "failed";
+      });
+    builder.addCase(fetchOrders.pending, (state) => {
+      state.orderStatus = "loading";
+    });
+    builder.addCase(fetchOrders.fulfilled, (state, action) => {
+      const ordersXano: OrderXano[] = action.payload;
+      ordersXano.forEach((order) => {
+        delete order["id"];
+        delete order["created_at"];
+      });
+      state.orders = ordersXano;
+      state.orderStatus = "completed";
+    });
+    builder.addCase(fetchOrders.rejected, (state) => {
+      state.orderStatus = "failed";
     });
   },
 });
 
 export const createOrder = createAsyncThunk<Order, void, { state: RootState }>(
   "orders/createOrder",
-  (_, thunkAPI) => {
-    const state = thunkAPI.getState() as RootState;
-    const currentBread = breadSelector(state);
-    const addCheese = addCheeseSelector(state);
-    const currentCheese = cheesesSelector(state);
-    const addMeat = addMeatSelector(state);
-    const currentMeat = meatsSelector(state);
-    const addDressing = addDressingSelector(state);
-    const currentDressing = dressingsSelector(state);
-    const currentVegetables = vegetablesSelector(state);
-    const addEgg = addEggSelector(state);
-    const currentEgg = eggsSelector(state);
-    const currentSpread = spreadSelector(state);
-    const currentServing = servingSelector(state);
-    const currentTopping = toppingSelector(state);
-    const currentName = nameSelector(state);
-    const currentDefaultName = defaultNameSelector(state);
-    const currentAdds = addToOrderSelector(state);
+  async (_, thunkAPI) => {
+    try {
+      const endpoint = "https://x8ki-letl-twmt.n7.xano.io/api:wGOItlwE/orders";
```

Comment:
do you use it?

Reply:
I don't know what do you mean.

Reply:
I have placed endpoint declaration closer to the fetch function where it's used.